meta-rz-bsp/dynamic-layers/meta-rz-panfrost/recipes-core/packagegroups/packagegroup-rz-tools-gpu.bbappend

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2024, Renesas Electronics Corp.
+#
+# SPDX-License-Identifier: MIT
+
+PACKAGE_ARCH = "${MACHINE_ARCH}"
+
+inherit packagegroup
+
+RDEPENDS:${PN} = " \
+	startup \
+"
```

Comment:
The contents of the bbappend file are appended to an underlying recipe. Bitbake will read `meta-rz-bsp/recipes-core/packagegroups/packagegroup-rz-tools-gpu.bb` and then append the contents of `meta-rz-bsp/dynamic-layers/meta-rz-panfrost/recipes-core/packagegroups/packagegroup-rz-tools-gpu.bbappend`, so the bbappend file only needs to contain the modifications to the recipe.

In this case we just want to add `startup` to `RDEPENDS:${PN}`, so we can just have the following using `+=`:

```
# Copyright (c) 2024, Renesas Electronics Corp.
#
# SPDX-License-Identifier: MIT

RDEPENDS:${PN} += " \
	startup \
"
```

Reply:
Done, good point! :D
Thanks.